lua/menu.lua

```diff
@@ -223,28 +220,18 @@ norns.menu.get_enc = function() return menu.penc end
 norns.menu.get_key = function() return menu.key end
 norns.menu.get_redraw = function() return menu.redraw end
 
-norns.scripterror = function(msg)
-  local msg = msg;
-  if msg == nil then msg = "" end
-  print("### SCRIPT ERROR: "..msg)
-  menu.errormsg = msg
-  menu.scripterror = true
-  menu.set_page(pHOME)
-  menu.set_mode(true)
-end
-
+-- init menu after script load
 norns.init_done = function(status)
   menu.set_page(pHOME)
-  if status == true then
-    menu.scripterror = false
+  if norns.script.errorstate == true then
     m.params.pos = 0
-    menu.set_mode(false)
+    menu.mode = true
```

Comment:
aside: maybe a more descriptive name?  from a quick scan, this looks odd next to `menu.set_mode(menu.mode)`.  what does setting mode to `true` mean?